### DISCUSSION TITLE
Getting a new System.DllNotFoundException error periodically with v9.0

### DISCUSSION THREAD

**Discussion by ericberman** [ImageMagick/ImageMagick#4789] [Help]
Anybody else seeing this?

Using C# and just upgraded via Nuget the other day to 9.0 Magick.NET.Core/Magick.NET-Q16-AnyCPU, and I'm getting periodic exceptions in this code:
MagickImage image = new MagickImage(s)
where "s" is a stream
Exception is:
System.TypeInitializationException: The type initializer for 'NativeMagickSettings' threw an exception. ---> System.DllNotFoundException: Unable to load DLL 'Magick.Native-Q16-x64.dll': The specified module could not be found

I can't reliably reproduce this, though.

**Comment by dlemstra** [comment]
More info can be found here instead: https://github.com/dlemstra/Magick.NET/issues/1121
